src/util/retry.test.js

```diff
@@ -33,16 +33,42 @@ describe('retry', () => {
 
   it('should retry the same function but stop when it has reached max attempts', async () => {
     const mock = jest.fn()
+    const error = new Error('some error')
```

Comment:
Can you add to the title of this test that it also checks if a specific error message gets passed?